requirements/requirements_test.txt

```diff
@@ -1,5 +1,7 @@
 ansys-mapdl-core>=0.60.4
-matplotlib
-pytest
-pytest-cov
-scipy
+matplotlib>=3.5.3
+pytest==7.2.1
+pytest-cov==4.0.0
+scipy>=1.7.3
+ipywidgets==8.0.4
+trame==2.2.6
```

Comment:
Thanks for pinning down the versions @clatapie - trame shouldn't be needed AFAIK. Otherwise, there is an error with ``pyvista``

Reply:
I had this [error ](https://github.com.mcas.ms/pyansys/pymapdl-reader/actions/runs/4104395946/jobs/7083618659#:~:text=ModuleNotFoundError%3A%20No%20module%20named%20%27trame%27) before adding it, that's why I made the change.